MeepMeeptest/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java-library'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+}
+repositories {
+    maven { url = 'https://jitpack.io' }
+    maven { url = 'https://maven.brott.dev/' }
```

Comment:
It looks like extra stuff unrelated to the ticket was checked in, please remove/undo